simulator/simulator.go

```diff
@@ -65,10 +55,7 @@ func startSimulator() error {
 		return xerrors.Errorf("create an etcd client: %w", err)
 	}
 
-	// need to sleep here to make all controllers create initial resources. (like "system-" priorityclass.)
-	time.Sleep(1 * time.Second)
```

Comment:
How can we know whether the kwok cluster is ready or not.

Reply:
I'd try to get the `kube-system namespace` to check whether cluster is ready.
I think it's better way as Namespace resource exist from the begining.
What do you think ?